src/odemis/driver/xt_client.py

```diff
@@ -376,6 +377,24 @@ def move_stage(self, position: Dict[str, float], rel: bool = False) -> None:
             self.server._pyroClaimOwnership()
             self.server.move_stage(position, rel)
 
+    def set_raw_coordinate_system(self, raw_coordinates:bool = True):
+        """
+        Read raw z coordinate or linked z coordinate when requesting stage coordinates
+        :param raw_coordinates: (bool) If True, request raw z stage coordinates, otherwise request linked Z coordiantes
+        """
+        try:
+            with self._proxy_access:
+                self.server._pyroClaimOwnership()
+                self.server.set_raw_coordinate_system(raw_coordinates)
+        except AttributeError:
```

Comment:
are you sure this will raise an `AttributeError`? I thought Pyro5 translated everything into an `OSError`

Reply:
I am not sure about it either, I have added it like below. In SEM class, I see mostly timeout errors and I see OS errors in Detector class
`        except OSError:
            if raw_coordinates:
                raise NotImplementedError("Ensure the version of Delmic XT Adapter is >= 1.12.0")`